Apps/VisionOS/README.md

```diff
@@ -0,0 +1,59 @@
+# VisionOS Sample App
+
+The VisionOS Sample App is a practical demonstration of how to integrate and utilize the Customer.io Data Pipelines iOS/Swift SDK.
+
+## Who Is This App For?
+
+This sample app is designed for VisionOS and iOS developers who are looking to get hands-on experience with Customer.io's code samples. It serves as a tool to validate, debug, or enhance their understanding of the different Customer.io APIs.
+
+## Run the App
+
+Before running the app, ensure you have one of the following prerequisites:
+* Apple Vision Pro simulator, which can be downloaded from Xcode.
+* An Apple Vision Pro device with developer mode enabled.
+
+To run the app, follow these steps:
+
+1. Clone the [customerio-ios](https://github.com/customerio/customerio-ios) repository by executing:
+   ```
+   git clone https://github.com/customerio/customerio-ios
+   ```
+2. Navigate to `<REPO_ROOT_DIR>/Apps/VisionOS` and open the `VisionOS.xcodeproj` file with Xcode.
+3. Ensure the `VisionOS` scheme is selected and set to run on either the simulator or your device.
+4. Click the run button in Xcode or press `CMD+R`.
+
+## How to Use the App
+
+### Prerequisites
+1. You must have a Customer.io account. Sign up for a free one if you haven't already.
+2. While a full workspace setup is not required for this sample app, you do need to set up an iOS Data Pipeline source and use its CDP API key to initialize the SDK. [Learn more](https://customer.io/docs/sdk/ios/getting-started/auth/#set-up-a-new-source).
+
+### MainScreen.swift
+`MainScreen.swift` contains the executable code samples that the app's UI will use during interaction. Place breakpoints in the `switch` case within this file to inspect any action performed by the app.
+
+### AppDelegate.swift
+The SDK is initialized when you enter the `CDP API key` and tap the `Initialize` button on the **Initialize** screen. In a production app, you would typically initialize the SDK in your `AppDelegate`. Therefore, you will find SDK initialization code in both `MainScreen.swift` and `AppDelegate.swift`.
```

Comment:
Since the most likely audience of this is a developer, should we allow them to set their CDP API Key via code? I think this might actually be easier than typing it via the "keyboard". Especially since the keyboard seems to be a bit of a pain with vision pro.

Reply:
They definitely can. If they want to, they will set it up in the app delegate as a hardcoded value instead of the value loaded from storage and based on the UI.